virttest/gluster.py

```diff
@@ -319,25 +320,47 @@ def create_gluster_uri(params, stripped=False):
     """
     vol_name = params.get("gluster_volume_name")
 
+    # Access gluster server by unix domain socket
+    unix_socket = os.path.exists(params.get("gluster_server", ""))
```

Comment:
Better to introduce a new param to represent the socket path required by unix transport.

Reply:
Introduced gluster_unix_socket.